server/repository/src/db_diesel/temperature_chart/temperature_chart.rs

```diff
@@ -23,30 +34,37 @@ impl<'a> TemperatureChartRepository<'a> {
         intervals: Vec<Interval>,
         temperature_log_filter: Option<TemperatureLogFilter>,
     ) -> Result<Vec<TemperatureChartRow>, RepositoryError> {
-        let mut query = TemperatureChart {
+        /*let mut query = temperature TemperatureChart {
+            intervals: intervals.clone(),
+        }*/
+        let mut query = temperature_chart::table {
             intervals: intervals.clone(),
         }
+        .group_by((temperature_chart::IntervalId, temperature_chart::SensorId))
```

Comment:
Had to move the `group_by` before the into_boxed(), otherwise I couldn't get it working...